src/chats/ChatsManager.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const messages = require('./messages');
+
+class ChatsManager {
+    /**
+     * @class ChatsManager
+     * @constructor
+     * @param {Object} store Store that implements interface for saving persistent data
+     * @param {Function} [mapper] Mapping function to select needed data from Telegram message entity when saving.
+     * Use tg-fluent's "createMessageMapper" function to create mapper.
+     */
+    constructor(store, mapper) {
+        if (typeof store === 'object') this.store = store;
+        else
+            throw Error(
+                'Store should be an object that implements interface for saving persistent data!'
+            );
```

Comment:
```suggestion
        if (typeof store !== 'object') throw Error(ERR_OBJ_EXPECTED);
        this.store = store;
```
Mode error message to constant.